sys/linux/socket_netlink_iscsi.txt.const

```diff
@@ -0,0 +1,7 @@
+# Code generated by syz-sysgen. DO NOT EDIT.
+arches = amd64
```

Comment:
The const files need to be generated for all arches:

$ syz-env make extract SOURCEDIR=/src/linux

where /src/linux is the latest linux-next checkout.